app/models/manageiq/providers/ansible_tower/automation_manager/inventory.rb

```diff
@@ -1,2 +1,2 @@
-class ManageIQ::Providers::AnsibleTower::AutomationManager::Inventory < ManageIQ::Providers::ExternalAutomationManager::InventoryGroup
+class ManageIQ::Providers::AnsibleTower::AutomationManager::Inventory < ManageIQ::Providers::ExternalAutomationManager::InventoryRootGroup
```

Comment:
The class you are trying to inherit from is missing, can you add it for External and Embedded as well?  Thanks for fixing this bug :+1: